algorithms/glued_trees/glued_trees.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>why specifically the 8 last bitstrings and not just the last one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I found that the trend is more clear when using multiple bitstrings as opposed to just one, and since we are modeling from t=2n-40 to t=2n+40, the expected spike for all 8 of the highest bitstrings is present within that range. I have added a paragraph to specify that.